docs/08-extensions.md

```diff
@@ -0,0 +1,129 @@
+# Extensions to OpenShift using OperatorHub
+
+OpenShift 4 has a slimmer base, with the ability to easily extend it with Operators for both cluster services (networking, storage, logging) and applications (databases, message queues, source control) for your developers to build applications with.
+
+OperatorHub is a feature built into your cluster to discover and install Operators on your cluster. OperatorHub is only available for cluster administrators. Once installed, these services are made available to users of the cluster through the Developer Catalog.
+
+Within OperatorHub, you will find three sets of Operators available to you:
+
+* Red Hat Products - licensed software that is tested extensively on OpenShift 4.
+* Certified Partners - partners that have certified their applications for OpenShift and have established a mutual support SLA with Red Hat, in order to provide you with the best experience.
+* Community - a set of curated Operators built by the Kubernetes community and work well with the Operator Lifecycle Management software already installed on your cluster.
+
+## Installing an Extension
+
+Installing your first Operator is best done through the user interface, but can also be driven by the command line. Navigate to "Catalog", then "OperatorHub".
+
+Search or browse to the Couchbase Operator, which is a powerful NoSQL database. The description lays out the notable features of the Operator. Go ahead and click "Install" to deploy the Operator on to the cluster.
+
+### Installation Method
+
+Operators can be installed for all Projects across the cluster or within specific namespaces. Not all Operators support each of these installation methods, but Couchbase does. Choose to install it for all Projects, so all users of the cluster can make Couchbase databases.
```

Comment:
For beta2, the `Couchbase` only support the single namespace.

Reply:
fixed already

Reply:
I'm sorry, but the document wasn't modified. Do you mean we changed the InstallMode of the `Couchbase` to follow this doc?